lib/menu.rb

```diff
@@ -0,0 +1,29 @@
+class Menu
+  attr_reader :menu
+
+  def initialize
+    @menu = menu
```

Comment:
Is this line required? Or, in other words, would your code behaviour change if you deleted the initialize method? If you're not sure, try it and see what happens